packages/gatsby-page-utils/src/watch-directory.ts

```diff
@@ -1,9 +1,13 @@
-const Promise = require(`bluebird`)
-const chokidar = require(`chokidar`)
+import * as chokidar from "chokidar"
 const { slash } = require(`gatsby-core-utils`)
 
-module.exports = async (path, glob, onNewFile, onRemovedFile) =>
-  new Promise((resolve, reject) => {
+export function watchDirectory(
```

Comment:
Why did you drop the `async`? Please restore that keyword. It won't change the runtime but it helps the tooling and the reviewers.

Reply:
Good catch